### ISSUE TITLE
Cannot clone a git repository that is hosted on-premise

### ISSUE THREAD

**Issue by msbmw** [getinsomnia/insomnia#5967]
### Expected Behavior

I want to be able to clone a git repository, that is hosted on-premise, e.g. `https://foo.bar.com/bitbucket/scm/org/repo.git`.

### Actual Behavior

If I try to clone this repository in Insomnia, I get the following error message:

<img width="368" alt="Screenshot 2023-05-16 at 12 47 09" src="https://github.com/Kong/insomnia/assets/99719282/f2eb9d68-f935-4773-906f-bd891cf86d51">

#5954 uses `hosted-git-info` for git url parsing, however it does **not** support on-premise hosted git repositories, see https://github.com/npm/hosted-git-info/issues/11

the following code will result in an empty string when trying to parse an on-premise git repository:
````ts
 repoSettingsPatch.uri = fromUrl(repoSettingsPatch.uri)?.https() || '';
````
see https://github.com/Kong/insomnia/blob/develop/packages/insomnia/src/ui/routes/git-actions.tsx#L327

### Reproduction Steps

1. Open the git clone UI: 
<img width="282" alt="Screenshot 2023-05-16 at 12 53 57" src="https://github.com/Kong/insomnia/assets/99719282/a42f863f-3e48-4ae2-bd43-d054d2edf1aa">

2. enter a git repo url that is not on one of the hosts that is supported by [`hosted-git-info` ](https://github.com/npm/hosted-git-info/blob/main/lib/hosts.js)

3. fill in the other fields
<img width="750" alt="Screenshot 2023-05-16 at 12 56 42" src="https://github.com/Kong/insomnia/assets/99719282/e0b0aa56-2fd4-4e7d-a333-5135929c0b95">

3. click on "Clone"

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2023.2.1

### What operating system are you using?

macOS

### Operating System Version

macOS 13.3.1

### Installation method

installed via brew

**Comment by filfreire** [comment]
@msbmw thanks for reporting this, we'll have the fix for it soon out. In the meantime please feel free to use the previous version, 2023.2.0 instead of 2023.2.1

**Comment by filfreire** [comment]
@msbmw fix is merged, will go out in upcoming 2023.2.2 patch
